File: Udemy One Week Python/exercise_files/section_14/beer.py
# WITH A FOR LOOP
for num_bottles in range(99,0,-1):
    print(f"{num_bottles} bottles of beer on the wall.")
    print(f"{num_bottles} bottles of beer.")
    if num_bottles == 1:
        print(f"take one down, pass it around,no more bottles of beer on the wall.")
    else:
        print(f"take one down, pass it around,{num_bottles - 1} bottles of beer on the wall.")
    print("*" * 50)

# WITH A WHILE LOOP
num_bottles = 99
while num_bottles > 0:
    print(f"{num_bottles} bottles of beer on the wall.")
    print(f"{num_bottles} bottles of beer.")
    if num_bottles == 1:
        print(f"take one down, pass it around,no more bottles of beer on the wall.")
    else:
        print(f"take one down, pass it around,{num_bottles - 1} bottles of beer on the wall.")
    print("*" * 50)
    num_bottles -= 1

File: Udemy_The_Complete_Java_Development_Bootcamp_Rayan_Slim/exercise_files/section 10/Beer.java
public class Beer {

  public static void main(String[] args) {
    // See detailed instructions on Learn the Part.
    for (int i = 99; i > 0; i--) {
      sing(i);
    }
  }

  public static void sing(int i) {
    System.out.println(
      i + " bottles of beer on thr wall, " + i + " bottles of beer!"
    );
    System.out.println(
      "take one down, pass it around " +
      (i - 1) +
      ", bottles of beer on the wall!"
    );
  }
}
